packages/runtime/container-runtime/src/test/runningSummarizer.spec.ts

```diff
@@ -32,7 +32,8 @@ class MockRuntime {
     ) { }
 }
 
-describe("Runtime", () => {
+// These tests don't pass, due to .only accidentally checked into summarizerHeuristic.spec.ts for a few weeks
+describe.skip("Runtime", () => {
```

Comment:
@kian-thompson FYI.  What's the status of fixing these?

Reply:
I haven't gotten to it due to other higher priorities

Reply:
K that's fine. FYI on this change then :)

Reply:
#11240